docs/sdk/trace.rst

```diff
@@ -8,6 +8,7 @@ Submodules
 
    trace.export
    trace.sampling
+   trace.random_ids_generator
```

Comment:
Adding it to the docs here for visibility. Also because both `sampling.py` and `export/` are added and they are in the same directory level.

Reply:
See above about including in the sdk.